From d702b3b574a65a8eeb0e6091f8a73ac7a3b97ae3 Mon Sep 17 00:00:00 2001
From: Gowtham Anandha Babu <gowtham.anandha.babu@intel.com>
Date: Wed, 6 Mar 2024 16:55:31 +0530
Subject: [PATCH] Add BT HFP support for TyP2

Send sco handle during BT HFP sco call for Intel Bluetooth AX210
bt card (TyphoonPeak - Typ2)

Tracked-On: OAM-121421
Signed-off-by: Gowtham Anandha Babu <gowtham.anandha.babu@intel.com>
---
 bluetooth/h4_protocol.cc | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/bluetooth/h4_protocol.cc b/bluetooth/h4_protocol.cc
index 1b29117..4463518 100644
--- a/bluetooth/h4_protocol.cc
+++ b/bluetooth/h4_protocol.cc
@@ -40,6 +40,7 @@
 #define INTEL_PID_9560 0x0aaa // 9460/9560 also know as Jefferson Peak (JfP)
 #define INTEL_PID_AX201 0x0026 // AX201 also know as Harrison Peak (HrP)
 #define INTEL_PID_AX211 0x0033 // AX211 also know as GarfieldPeak (Gfp)
+#define INTEL_PID_AX210 0x0032 // AX210 also know as TyphoonPeak (TyP2)
 
 #include <errno.h>
 #include <fcntl.h>
@@ -88,7 +89,8 @@ bool H4Protocol::IsIntelController(uint16_t vid, uint16_t pid) {
                                 (pid == INTEL_PID_9260)||
                                 (pid == INTEL_PID_9560)||
                                 (pid == INTEL_PID_AX201)||
-                                (pid == INTEL_PID_AX211)))
+                                (pid == INTEL_PID_AX211)||
+                                (pid == INTEL_PID_AX210)))
         return true;
     else
 	return false;
-- 
2.17.1

